IFC4x3/PropertyEnumerations/b/PEnum_BoreholeState.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="PEnum_BoreholeState" UniqueId="84d3d099-2c7c-40b5-97f4-514b9ecd836a">
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PEnum_BoreholeState" Name="PEnum_BoreholeState" UniqueId="84d3d099-2c7c-40b5-97f4-514b9ecd836a">
```

Comment:
This one requires an encoded guid in id. Re-serializing with IfcDoc would probably make sense.

Reply:
I will merge and regenerate if necessary.